### PR TITLE
Checks for calls that depend on dotted paths.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by Horneth** [comment]
👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/227/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/dot_dir_stuck_running.test [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+testFormat: workflowsuccess
+
+files {
+  wdl: dot_dir_stuck_running/dot_dir_stuck_running.wdl
+}
+
+metadata {
+  workflowName: dot_dir_stuck_running
+  status: Succeeded
+  "outputs.dot_dir_stuck_running.out" : "hi"
```
Comment (cjllanwarne) [inline src/main/resources/standardTestCases/dot_dir_stuck_running.test]: Maybe also verify the `verify_not_stuck.out`?

**Comment by cjllanwarne** [comment]
I found the naming somewhat confusing, but otherwise 👍 

[Approved with PullApprove](https://pullapprove.com/broadinstitute/centaur/pull-request/227/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=broadinstitute/centaur)

File: src/main/resources/standardTestCases/dot_dir_stuck_running.test (hunk 1 above)
Comment (kshakir) [reply src/main/resources/standardTestCases/dot_dir_stuck_running.test]: Good point, but I don't know how to verify file paths. Mainly listed the path as a visual debugging tool.

**Comment by kshakir** [comment]
Tested here: https://travis-ci.org/broadinstitute/cromwell/jobs/262056622#L442

**Comment by cjllanwarne** [comment]
👍 again

**Pull request merged**